### ISSUE TITLE
Bookmark is removed when first character of line is deleted

### ISSUE THREAD

**Issue by izuzak** [atom/bookmarks#28]
:tv:

![book](https://cloud.githubusercontent.com/assets/38924/7301046/608e6d98-e9e0-11e4-993a-dfc4d90230e4.gif)

(that's in safe mode :point_up: )

@benogle just wondering if that's expected or not (looks a bit strange to me, and I'm not seeing that behavior in SublimeText). When exactly should bookmarks be deleted when editing text in the editor (what are the situations in which we _want_ the bookmarks to be deleted as a result of text changes)?

**Comment by dander401** [comment]
This also happens when code is reformatted

**Comment by dwelle** [comment]
@dander401, yep - looks like it doesn't work well with [JSCS](https://github.com/AtomLinter/linter-jscs)'s fix on save feature removes all bookmarks. I thought I was going mad, my bookmarks randomly disappearing.

**Comment by mikedaly** [comment]
Ah, so that's what I've been seeing. Here's the associated issue in linter-jscs: 
https://github.com/AtomLinter/linter-jscs/issues/136
